website/docs/d/multicast_pool.html.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: "aci"
+page_title: "ACI: aci_multicast_pool"
+sidebar_current: "docs-aci-data-source-multicast_pool"
+description: |-
+  Data source for ACI Multicast Address Pool
+---
+
+# aci_multicast_pool #
+
+Data source for ACI Multicast Address Pool
+
+## API Information ##
+
+* `Class` - fvnsMcastAddrInstP
+* `Distinguished Name` - uni/infra/maddrns-{name}
+
+## GUI Information ##
+
+* `Location` - 
```

Comment:
GUI Location is missing.

Reply:
* `Location` - Fabric -> Access Policies -> Pools -> Multicast Address -> Address Blocks

Reply:
ok

Reply:
ok